api/app/controllers/api/v1/tenants_controller.rb

```
@@ -0,0 +1,79 @@
+              }
+            }
+          }, status: :ok
+        else
+          render json: { error: 'No tenant found' }, status: :not_found
+        end
+      end
+
+      private
+
+      def require_super_admin!
+        unless current_admin.super_admin?
```

RuboCop (1.81.1)

[convention] Use a guard clause (`return if current_admin.super_admin?`) instead of wrapping the code inside a conditional expression.

(Style/GuardClause)